go/Makefile

```diff
@@ -1,3 +1,5 @@
+BAZEL := $(shell bash -c "which bazel")
```

Comment:
probably worth a small comment pointing out the windows make binary resolution bug, or someone in the future might revert this thinking "why not use `bazel` directly?"